server/member/TeamMember.java

```diff
@@ -0,0 +1,118 @@
+package com.objectcomputing.checkins.services.guild.member;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.micronaut.data.annotation.AutoPopulated;
+import io.micronaut.data.annotation.TypeDef;
+import io.micronaut.data.model.DataType;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.annotation.Nullable;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Table(name = "guildMembers")
```

Comment:
guildMembers

Reply:
Looks like a lot of unwanted changes got brought in somehow. Can you fix this?

Reply:
Removed the unwanted changes brought from my new issue that I am working on. Fixed. Thanks.